controlservice.proto

```diff
@@ -60,7 +57,8 @@ service ControlService {
   rpc Log(LogMessage) returns (LogResponse) {}
   rpc Quit(QuitMessage) returns (QuitResponse) {}
   rpc SwitchPhase(SwitchPhaseMessage) returns (SwitchPhaseResponse) {}
-  rpc GetPackageNode(PackageRequest) returns (PackageNode) {}
+  rpc GetPackageNode(PackageNode) returns (PackageNode) {}
+  rpc GetProjectNode(ProjectNode) returns (ProjectNode) {}
```

Comment:
We want to get away from having node based functions in the control service. Please put them into the Build service.

Reply:
We need to be able to access these functions in all phases. Right now GetPackageNode is used in analysis phase and reporting phase.